### DISCUSSION TITLE
Do you have better idea for repository name than "jargon" ?

### DISCUSSION THREAD

**Discussion by ccoVeille** [jargonLint/jargonLint#2] [General]
I initiated the project with the name "jargon" because it's an English and a French word (I'm French) and I like how old-fashioned it sounds. 

I would like everyone to suggest name if you have read #1 and you have better idea/suggestion

Thanks

**Comment by ccoVeille** [comment]
jargon (keep jargon)

**Comment by ccoVeille** [comment]
jargon-list

**Comment by ccoVeille** [comment]
jargon-database

**Comment by ccoVeille** [comment]
jargon-lint

**Comment by ccoVeille** [comment]
jargon-terms

**Comment by ccoVeille** [comment]
spellverse

**Comment by alexvonme** [comment]
Can we close this for now @ccoVeille?
